scripts/stablex/utilities.js

```diff
@@ -31,12 +32,47 @@ const addTokens = async function (token_addresses, web3, artifacts) {
   }
 }
 
+const depositTokens = async function (token_address, depositor, amount, artifacts) {
+
+  const StablecoinConverter = artifacts.require("StablecoinConverter")
+  const instance = await StablecoinConverter.deployed()
+  const ERC20 = artifacts.require("ERC20.sol")
+
+  const token = await ERC20.at(token_address)
+  const depositor_balance = (await token.balanceOf.call(depositor))
+  if (depositor_balance.lt(amount)) {
+    console.log(`Error: Depositor has insufficient balance ${depositor_balance} < ${amount}.`)
+  }
+
+  const allowance = (await token.allowance.call(depositor, instance.address)).toString()
+  if (allowance < amount) {
+    await token.approve(instance.address, amount, { from: depositor })
+  }
+
+  return await instance.deposit(token_address, amount, { from: depositor })
+
+}
+
+const placeOrder = async function (buyToken, sellToken, account, validFor, minBuy, maxSell, artifacts) {
+  const { sendTxAndGetReturnValue } = require("../../test/utilities.js")
+
+  const StablecoinConverter = artifacts.require("StablecoinConverter")
+  const instance = await StablecoinConverter.deployed()
+
+  const batch_index = (await instance.getCurrentBatchId.call()).toNumber()
+  const valid_until = batch_index + parseInt(validFor)
+
+  await sendTxAndGetReturnValue(instance.placeOrder, buyToken, sellToken, valid_until, minBuy, maxSell, { from: account })
```

Comment:
Is there any reason we don't just use `instance.placeOrder` here? I don't see us storing or "using" the return values here.